src/Nest/XPack/Graph/Explore/Request/GraphExploreControls.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Linq.Expressions;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	public interface IGraphExploreControls
+	{
+		[JsonProperty("use_significance")]
+		bool? UseSignificance { get; set; }
+
+		[JsonProperty("sample_size")]
+		int? SampleSize { get; set; }
+
+		[JsonProperty("timeout")]
+		Time Timeout { get; set; }
+
+		[JsonProperty("sample_diversity")]
+		SampleDiversity SampleDiversity { get; set; }
+	}
+
+	public class GraphExploreControls : IGraphExploreControls
+	{
+		[JsonProperty("use_significance")]
```

Comment:
Do we need the `JsonProperty` attributes on the concrete type and the interface?